profiles/kentik_snmp/checkpoint/checkpoint-firewall.yml

```diff
@@ -151,3 +151,15 @@ metrics:
     symbol:
       OID: 1.3.6.1.4.1.2620.1.1.25.4.0
       name: fwPeakNumConn
+- MIB: CHECKPOINT-MIB
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.2620.1.6.7.9.1.1.1
+          name: powerSupplyIndex
+        tag: sensor_index
+    symbols:
+      - OID: 1.3.6.1.4.1.2620.1.6.7.9.1.1.2
+        name: powerSupplySensorStatus
+    table:
+      OID: 1.3.6.1.4.1.2620.1.6.7.9
+      name: powerSupplySensorTable
```

Comment:
@montecas - thanks for the submission; this section has some indentation and missing conversion errors that need to be resolved, and we'd like to keep the order of elements in line with the rest of the repo's patterns

```
table
symbols
metric_tags
```

suggested change:

```yaml
  - MIB: CHECKPOINT-MIB
    table:
      name: powerSupplySensorTable
      OID: 1.3.6.1.4.1.2620.1.6.7.9
    symbols:
      # this OID returns a 'DisplayString', we need to convert it to a metric
      - name: powerSupplySensorStatus
        OID: 1.3.6.1.4.1.2620.1.6.7.9.1.1.2
        conversion: to_one
```

1. indent added
2. re-order sections
3. add conversion to handle OID that returns a string value
4. remove index from `metric_tag` (we always collect the index, you don't need to call it out)

Reply:
This works for me, thank you for the feedback. Shall I update my PR with this or will you pull in the suggested changes?

Reply:
please make the updates; thanks!